Terminal.Gui/Views/ScrollView.cs

```diff
@@ -134,6 +136,26 @@ public ScrollView ()
                        };
     }
 
+    private void Application_UnGrabbedMouse (object sender, ViewEventArgs e)
+    {
+        var parent = e.View is Adornment adornment ? adornment.Parent : e.View;
+
+        if (parent is { })
+        {
+            var supView = parent.SuperView;
+
+            while (supView is { })
+            {
+                if (supView == _contentView)
+                {
+                    Application.GrabMouse (this);
+                }
```

Comment:
Are you sure there shouldn't be a `break;` here to exit the loop?

Reply:
You are absolutely right. When I did that I think to my self, don't forget to break the loop if found it. Damn and I forgot. Thanks.